src/camera.rs

```diff
@@ -11,6 +11,8 @@ use vecmath::{
 };
 use vecmath::col_mat4_mul as mul;
 use vecmath::consts::Radians;
+use cgmath::{ Quaternion, Rotation, FixedArray, BaseFloat };
+use cgmath::Vector3 as CgVector3; 
```

Comment:
Is there a good reason to not switch completely to cgmath in this lib? Quaternions are kinda nice for a camera library.